docs/source/examples/vision/guides/formatters_validating.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>Explain we use the default state not because it's correct, but to showcase the <code>validate_formatters</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>